2-multitenant/README.md

```diff
@@ -1,2 +1,54 @@
 # Multitenant Infrastructure phase
-This phase deploys multitenant infrastruture using the multitenant infrastructure pipeline.
+
+## Purpose
+
+This phase deploys the per-environment multitenant resources deployed via the multitenant infrastructure pipeline.
+
+The following resources are created:
+- GCP Projects
+- GKE cluster(s)
+- Cloud SQL PostgreSQL
+- Cloud Endpoint
+- Cloud Armor
+- IP addresses
```

Comment:
What are those IPs for?

If they are part of the DB, I'd just call it as "DB and resources for the DB."

Reply:
Switching to this output to an array (name : self_links) for clarity.